.github/workflows/e2ev2-provision-test.yaml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Print test logs
         shell: bash
         run: cat testing/e2e/job-app-routing-operator-e2e.log
-        if: always() && steps.test.outcome == 'failure'
+        if: always() && (steps.test.outcome == 'failure' || steps.test.outcome == 'success')
```

Comment:
What are all the options for the outcome?

Reply:
failure, success, skipped. The intent is to only run this if the previous command ran (because that's what created the log we are `cat`ing). 

I will note that the `always()` function is special too. It's not like `true` like you might think (that would be redundant to have a `true` && (other expression)). It makes it so this step will run even if the job's overall status is cancelled (which would happen if the previous step failed).

Reply:
then id nit this to be swapped to `steps.test.outcome != 'skipped'`

Reply:
https://docs.github.com/en/actions/learn-github-actions/contexts#steps-context

there's also cancelled.

Reply:
Forgot about that one. I'd rather be explicit about what we accept. Doing the inverse would make this incorrect if they add new statuses in the future.